packages/next-app/src/app/internalmanager/layout.tsx

```diff
@@ -1,28 +1,40 @@
 "use client";
 
+import { Route } from "next";
 import * as React from "react";
 import { useNetwork } from "wagmi";
 
 import balancerSymbol from "#/assets/balancer-symbol.svg";
-import { Header } from "#/components/Header";
+import { Header, HeaderNetworkMismatchAlert } from "#/components/Header";
 import { CheckSupportedChains } from "#/components/SupportedChain";
 import { InternalManagerProvider } from "#/contexts/InternalManagerContext";
-import { NetworksContextProvider } from "#/contexts/networks";
+import { getNetwork, NetworksContextProvider } from "#/contexts/networks";
 import { chains } from "#/wagmi/client";
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   const { chain } = useNetwork();
+
+  const network = getNetwork(chain?.name);
+
   return (
     <NetworksContextProvider>
       <div className="flex h-full flex-col">
+        <HeaderNetworkMismatchAlert />
         <Header
-          linkUrl={"/internalmanager"}
+          linkUrl={`/internalmanager/${network}` as Route}
```

Comment:
if eslint is not recognizing this as a route there's probably something weird with the `network` type 🤔

Reply:
probably related to nextjs' typedRoutes -- you can disable that in nextjs.config.js if it's getting in the way